root/static/scripts/guess-case/modes.js

```diff
@@ -137,19 +137,24 @@ function runFixes(is, fixes) {
 const DefaultMode = {
   description: '',
 
+  fixEnglishApostrophes() {
```

Comment:
Since this returns a boolean and doesn't perform an action, I think something like `shouldFixEnglishApostrophes` would be more clear, especially since we have other methods starting with "fix" that actually perform actions (`fixVinylSizes`, `fixEnglishKeyNames`, etc.).
```suggestion
  shouldFixEnglishApostrophes() {
```

Though this method also seems a bit weird since it's basically "is this mode English" and not really to do with apostrophes. I'd think we should just have a `gc.mode.name === 'english'` check or something where it matters.